### PR TITLE
ROIValues and compute_total_ROI_values exposed to python

### PR DESCRIPTION
**Pull request by robbietuk**
I have tested on my machine and I have access to `ROIValues` class and `compute_total_ROI_values`.

### CODE REVIEW

**Comment by robbietuk** [comment]
Python example script outputs the following to console.

```
python % python ROI_demo.py

Running ROI_demo.py
Loading init.hv as image...
Populating image with random values...
Constructing a EllipsoidalCylinder as the Region Of Interest (ROI)...
Setting the number of samples of each axis to be 10...
Computing ROI value...

INFO: Number of voxels recomputed with finer sampling : 70
ROI mean value = 0.4850385785102844
ROI stddev value = 0.2259778380393982
```

with the `INFO: Number of voxels recomputed with finer sampling : 70` being output by STIR

**Comment by robbietuk** [comment]
I have tested against the STIR executable `list_ROI_values` using parameter files and I get the same results via python. In terms of functionality, the PR is ready to merge.

File: examples/python/ROI_demo.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# Construct an example shape
+print("Constructing a EllipsoidalCylinder as the Region Of Interest (ROI)...")
+ROI_shape = stir.EllipsoidalCylinder(10, 5, 4, stir.FloatCartesianCoordinate3D(0, 0, 0))
+
+
+
+# number_of_sample needs to be a 3D Int CartesianCoordinate
+n = 10
+print(f"Setting the number of samples of each axis to be {n}...")
+number_of_sample = stir.IntCartesianCoordinate3D(n, n, n)
```
Comment (KrisThielemans) [inline examples/python/ROI_demo.py:44]:
```suggestion
number_of_samples = stir.IntCartesianCoordinate3D(n, n, n)
```

File: examples/python/ROI_demo.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+# Do ROI evaluation
+print("Computing ROI value...")
+ROI_eval = stir.compute_total_ROI_values(image, ROI_shape, number_of_sample)
```
Comment (KrisThielemans) [inline examples/python/ROI_demo.py:48]:
```suggestion
ROI_eval = stir.compute_total_ROI_values(image, ROI_shape, number_of_samples)
```

File: src/swig/stir.i [hunk 3]
```diff
@@ -1471,6 +1476,12 @@ namespace stir {
 %include "stir/Shape/EllipsoidalCylinder.h"
 %include "stir/Shape/Box3D.h"
 
+// ROIValues class and compute compute_ROI_values
+%shared_ptr(stir::ROIVales)
```
Comment (KrisThielemans) [inline src/swig/stir.i:1480]:
```suggestion
%shared_ptr(stir::ROIValues)
```

File: src/swig/stir.i [hunk 4]
```diff
@@ -1471,6 +1476,12 @@ namespace stir {
...
+%shared_ptr(stir::compute_total_ROI_values)
```
Comment (KrisThielemans) [inline src/swig/stir.i:1481]:
```suggestion
```
this is a function, not an object. (surprised that SWIG doesn't complain)

**Review by KrisThielemans — CHANGES_REQUESTED** [review]
great. Please add something to `release_5.0.html`

**Pull request merged**